timesketch/frontend/src/components/Sketch/TimelineListItem.vue

```diff
@@ -135,6 +135,32 @@ limitations under the License.
       Added {{ timeline.updated_at | moment("YYYY-MM-DD HH:mm") }}
       <span class="tag is-small" :title="meta.stats[timeline.searchindex.index_name]['count'] + ' events in index'">{{ meta.stats[timeline.searchindex.index_name]['count'] | compactNumber }}</span>
     </span>
+
+    <div v-if="timelineStatus === 'ready'" class="is-size-6 small-top-margin">
+      <a @click="toggleExpansion(timeline)">
+        <span class="icon">
+            <i :class="[isExpanded(timeline) ? 'fas fa-minus-circle' : 'fas fa-plus-circle']"></i>
```

Comment:
What do you think of putting the expand/collapse control on the same line as `Added 2020-06-05 08:50 ...`? That way we don't get a new line after each timeline in the list.

Reply:
@berggren - how about this?
![TS-removed-whitespace](https://user-images.githubusercontent.com/1036124/90371941-3392e000-e05f-11ea-9a40-6c1bc8133fa2.png)

Reply:
Thanks Johan! As discussed on the call, I've implemented it as a button on the right side instead of a separate text below.